recipes/libcurl/all/conanfile.py

```diff
@@ -353,6 +354,11 @@ def _generate_with_autotools(self):
             env = VirtualRunEnv(self)
             env.generate(scope="build")
 
+        # Before 7.86.0, enabling unix sockets configure option would fail on windows
+        # It was fixed with this PR: https://github.com/curl/curl/pull/9688
+        if self._is_mingw and Version(self.version) < "7.86.0":
+            self.options.with_unix_sockets = False
```

Comment:
Not like this please. Remove this option in `config_options()` if Windows, then refer to it with `self.options.get_safe("with_unix_sockets", False)`

Reply:
Otherwise you can just backport the patch to all versions before 7.86.0

Reply:
Can I set it to false instead of deleting it?

Reply:
@SpaceIm, backporting sounds like a good idea. I don't have the capacity to do it that for all 9 versions before 7.86 at the moment. Maybe we can add an issue about it?